composer.json

```
@@ -27,16 +28,16 @@
         "dg/bypass-finals": "*"
     },
     "require": {
-        "php": "^7.4 || >=8.1",
+        "php": ">=7.4 || >=8.1",
```

But we know it does NOT work for PHP 8.0 and only supports 8.1 and higher...

```suggestion
        "php": "^7.4 || >=8.1",
```

If we want to make it installable on PHP 8.0 there's no need to have an union of version ranges there at all:

```suggestion
        "php": ">=7.4",
```

---

We do not want to support php 8.0.

I think that @dominikvoda was aiming for consistency between sides of the sides of the `||`, even though there is no practical difference, unless php folks surprise us with release of php 7.5.. :smile:

---

There is a large practical difference, because 8.0 >= 7.4

---

You are right, I have reverted the change :+1: